.github/workflows/test.yml

```diff
@@ -142,5 +138,8 @@ jobs:
     - name: Remove compiled application files
       run: mix clean
 
+    - name: Compile dependencies
+      run: mix compile
```

Comment:
Again, any context you can provide on why this step was added would be useful for reviewers.

Reply:
Described in the previous comment.